### ISSUE TITLE
Advice: When Put The Item to the Buying Slot, It Shouldn't Take It From Me

### ISSUE THREAD

**Issue by ghost** [kamildanak/VendingBlock#16]
for example, i am buying 1 diamond for 2 emeralds. i put 2 emeralds in the selling slot, and put 1 diamond to the buying slot. but it took my diamond, and i need that diamond too. so it should just copy the item, not take it from me. can you do that? it would be perfect. thanks.

**Comment by kamildanak** [comment]
You can use Advanced Vending Block that way, put 2 emeralds in the selling slot, and select diamond with select button.

**Comment by ghost** [comment]
yes i know advanced vending block, but normal vending block can has this feature... it would be pretty. such as the Forestry carpenter, you just put the item, it copied it, the item still in your hand/cursor.

**Comment by Xandermud** [comment]
I would love it if either the normal vending block accepted ghost items (as suggested above) or if the advanced vending block allowed you to select a specific credit amount if you have EnderPay installed (maybe by including items in your inventory along with the creative inventory menu). Currently, if a player wants to sell an item for let's say 200 credits, that player needs to already have 200 credits AND they need to give up said credits for the duration of the sale (potentially forever, if the bank notes are set to expire after a certain number of days).
